### PR TITLE
Add key binding for 'delete'

### LINKED ISSUE #98: keyboard command to delete image
Via @steviepubliclab -- 

> I've had the problem multiple times where I'm running a workshop, or just trying to work in MK and we can't get the toolbar to show up. I can usually work around the with the commands, but there isn't a command for deleting an image which always seems to be a problem. Can we make one?

### PR DESCRIPTION
**Pull request by sashadev-sky**
fixes #98 

- [x] passes tests 
- [x] no merge conflicts
- [x] unique branch name

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@publiclab/reviewers 
@jywarren I created two keybindings because 'delete' is different on macs than specified by JS character codes. With this change both backspace and delete will delete the image. See my notes in the PR! Is that okay? 

also I couldn't find any information on this, is there a difference between firing a 'delete' event vs. a 'remove' event? I saw both used in the code. And why do we fire instead of just calling `.remove()`?

File: src/edit/DistortableImage.Edit.js [hunk 1]
```diff
@@ -91,7 +93,12 @@ L.DistortableImage.Edit = L.Handler.extend({
 		L.DomEvent.off(window, 'keydown', this._onKeyDown, this);
 
 		overlay.fire('deselect');
-	},
+  },
+  
+  confirmDelete: function () {
+    return window.confirm("Are you sure you want to delete?");
```
Comment (jywarren) [inline src/edit/DistortableImage.Edit.js]: nice!

**Comment by jywarren** [comment]
Hi, i'm not sure, to tell the truth - this was a long time ago. I would expect we should: a) look in the tests to see the preferred method, b) see if we can read the blame view for the deletion/removal code to trace what was done around that time, and c) see if we can write a test that uses a simpler method and confirm that it works just as well.

Thanks, @sashadev-sky !

**Comment by sashadev-sky** [comment]
@jywarren I was just wondering for myself, I don’t see anything wrong with the code since it’s consistent with it’s use of ‘fire’, changing to remove() would be inconsistent. If you want simpler code all around, though, I will look and see what I can refactor! 

As for my changes you are happy with those?

**Comment by jywarren** [comment]
yes thery're great, TY!

OK, so this is ready to merge then? Thanks!!!

**Comment by sashadev-sky** [comment]
@jywarren sorry for the delayed reply! I have been transferring to a new computer. 

Yes it mirrors the delete functionality using the toolbar, so it doesn't address other issues like deleting the toolbar as well on delete. But I figured that is something we will address separately for both at once?

**Comment by jywarren** [comment]
Fantastic, thank you!!!

**Pull request merged**